### ISSUE TITLE
Tutorial note buttons lack visible focus indicators

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#267]
## Steps to reproduce

1. Log in and open an assignment.
2. When the tutorial tooltip appears, try to focus the `I understand` button to dismiss it using only the keyboard.

## Expected outcome

Interactive elements such as buttons should have a visible focus indicator as per [WCAG success criterion 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html). Not all people can use a pointer input device, so meeting this success criteria enables those people to interact with the app. Here's a GIF showing this happening on Malmö stad's website.

<img
 alt="GIF showing visible focus indicator on Malmö stad's website"
 width="512"
 src="https://user-images.githubusercontent.com/566159/191342880-f477332b-c690-467d-8de1-e6495cfdf7fa.gif">
 
## Actual outcome

There's no focus indicator on the tooltip's buttons so it's impossible for a keyboard-only user to determine when they receive focus.

![Devtools screenshot showing the :focus element state enabled with no visible indicator](https://user-images.githubusercontent.com/566159/191564817-c88f3368-6f30-447a-be12-dbee10b5a653.png)

## Severity

The success criterion covering this issue is WCAG level AA – the middle level. This is the conformance level that most organizations aim for. The lack of a focus indicator effectively makes these buttons inoperable, making the tooltips permanent and potentially obscuring other UI components, so the severity of this one is high.

## Recommendation

The hover state for this component could be easily reused as a focus indocator by changing the selector from `&:hover` to `&:hover, &:focus`.

![Hover state with nice rounded pill button](https://user-images.githubusercontent.com/566159/191565163-288822c7-8f96-4be8-926a-8d3363e30543.png)
